packaging/CPackOptions.cmake

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019-2019 the openage authors. See copying.md for legal info.
+
+# Keeps all required CPack configuration for the Generators in one place and
+# CPack calls this file for every generator, keep this in mind, if you use
+# variables or functions outside of the conditional branch
+
+
+########################################################################
+# Continuous Integration
+
+# Filename addition for architecture, set from environment variables
+set(CPACK_PACKAGE_ARCHITECTURE "-$ENV{TARGET_PLATFORM}")
+
+# Filename addition for nightly-builds, set from environment variables
+if(DEFINED ENV{IS_NIGHTLY})
+    set(PACKAGE_NIGHTLY "_NIGHTLY")
+else()
+    set(PACKAGE_NIGHTLY "")
+endif()
+
+
+if(CPACK_GENERATOR MATCHES "NSIS")
+    # Explanations to the following commands you can find here:
+    # https://cmake.org/cmake/help/latest/cpack_gen/nsis.html#cpack_gen:CPack%20NSIS%20Generator
+
+    ########################################################################
+    # Filename
+
+     set(CPACK_PACKAGE_FILE_NAME "${CPACK_PACKAGE_NAME}-v${CPACK_PACKAGE_VERSION}${CPACK_PACKAGE_ARCHITECTURE}-installer${PACKAGE_NIGHTLY}")
+
+
+    ########################################################################
+    # Installer
+
+    set(CPACK_NSIS_MODIFY_PATH "OFF")
+    set(CPACK_NSIS_ENABLE_UNINSTALL_BEFORE_INSTALL "ON")
+    set(CPACK_NSIS_PACKAGE_NAME "${CPACK_PACKAGE_NAME} v${CPACK_PACKAGE_VERSION}")
+    set(CPACK_NSIS_DISPLAY_NAME "${CPACK_PACKAGE_NAME} v${CPACK_PACKAGE_VERSION}")
+    set(CPACK_NSIS_URL_INFO_ABOUT "https://github.com/SFTtech/openage")
+    set(CPACK_NSIS_HELP_LINK "https://github.com/SFTtech/openage/blob/master/doc/troubleshooting.md")
+
+    # TODO: create welcome page
+    #set(CPACK_RESOURCE_FILE_WELCOME "")
+
+
+    ########################################################################
+    # Shortcuts
```

Comment:
Same here, I even thought about compiling that bat into an exe just to use this ... but doesn't make sense. As soon as we have a launcher, it should be referenced here.